### ISSUE TITLE
Private tag included in functions name when renaming

### ISSUE THREAD

**Issue by arthuro555** [4ian/GDevelop#2494]
## Describe the bug
When renaming a private extension function, the ` (private)` tag is included in the text to rename

## To Reproduce
1. Go to an extension function
2. Make it private
3. Try renaming it
4. See issue:
![image](https://user-images.githubusercontent.com/19349038/113212463-1837b680-9277-11eb-9c77-842892de28b0.png)
![image](https://user-images.githubusercontent.com/19349038/113212487-1f5ec480-9277-11eb-8d1c-a941e0b157fd.png)

* Please include a link to a game if possible!
* If applicable, add screenshots to help explain your problem.

## Other details
* GDevelop beta 108
* Windows 10

**Comment by Bouh** [comment]
An lock icon or an eye icon coulb be better to display.

**Comment by 4ian** [comment]
Yeah the "abuse" of the label is not great, would be cool to extend the list item to allow to display an icon :)

**Comment by Bouh** [comment]
There is an thumbnail possible on virtualized item in list.
![image](https://user-images.githubusercontent.com/1670670/113218024-1eca2c00-927f-11eb-9f9e-a9ec27977205.png)
Maybe the icon have to be moved on right, and the icon are not great it was for example.
Idk if we can pass an icon component, but we can use image easily!

**Comment by tristanbob** [comment]
I just came here to report this, but @arthuro555 did a great job finding this in March.   :)
 
I like the idea of using an icon (with a mouse-over description) to indicate a private function.   Perhaps we can use an icon that represents visibility (since it is hidden from the end-users), like this one:

https://bit.dev/mui-org/material-ui-icons/visibility-off

**Comment by Bouh** [comment]
An easy quick win for you @ClementPasteau !
